# There is in fact no LGPL or GPL code in this package. It is just used with some.
# Corrected package.xml accordingly.

Index: ros-python-qt-binding/package.xml
===================================================================
--- ros-python-qt-binding.orig/package.xml
+++ ros-python-qt-binding/package.xml
@@ -4,7 +4,7 @@
   <description>
     This stack provides Python bindings for Qt.
     There are two providers: pyside and pyqt.  PySide is released under
-    the LGPL.  PyQt is released under the GPL.
+    the LGPL.  PyQt is released under the GPL. This package is BSD.
 
     Both the bindings and tools to build bindings are included from each
     available provider.  For PySide, it is called "Shiboken".  For PyQt,
@@ -16,8 +16,6 @@
   </description>
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>BSD</license>
-  <license>LGPL</license>
-  <license>GPL</license>
 
   <url>http://ros.org/wiki/python_qt_binding</url>
   <author>Dave Hershberger</author>
